pkg/controller/v1alpha1/inferencegraph/raw_ig.go

```diff
@@ -76,6 +84,7 @@ func createInferenceGraphPodSpec(graph *v1alpha1api.InferenceGraph, config *Rout
 			},
 		},
 		Affinity:                     graph.Spec.Affinity,
+		ServiceAccountName:           "default",
```

Comment:
Isn't better to name something else?
like `ig-sa`, wdyt?

Reply:
also, where is it used?

Reply:
I had to do this to be backwards-compatible, so that IGs that don't have auth-enabled are using the default SA (like when not specifying it).

This is covering the case for _disabling_ auth on an IG that already has it enabled. Without this, the SA will persist in the pod.